CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Changed
+
+- Make proxyLogDirectory a setting in the vscode extension rather than using the default for vscode,
+  and only send the --log-dir argument to coder/coder if the value is set.
```

Comment:
It could be helpful to add some description on why one might want to use the flag, and I think `--log-dir` could be considered an internal implementation detail so maybe we do not need to mention it.  Maybe something like:

```suggestion
- Previously, the extension would always log SSH proxy diagnostics to a fixed
  directory. Now this must be explicitly enabled by configuring a new setting
  `coder.proxyLogDirectory`. If you are having connectivity issues, please
  configure this setting and gather the logs before submitting an issue.
```